src/components/Garden/Feed/Delegation.js

```diff
@@ -0,0 +1,160 @@
+import React, { useCallback } from 'react'
+import { useHistory } from 'react-router'
+import {
+  Box,
+  Button,
+  EthIdenticon,
+  GU,
+  Link,
+  LoadingRing,
+  shortenAddress,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
+import useProfile from '@hooks/useProfile'
+import { useSupporterSubscription } from '@hooks/useSubscriptions'
+import { useWallet } from '@providers/Wallet'
+
+function Delegation({ onRemoveDelegate, onSetDelegate }) {
+  const { account } = useWallet()
+  const theme = useTheme()
+  const [supporter, loading] = useSupporterSubscription(account)
+
+  return (
+    <Box>
+      {loading ? (
+        <LoadingRing />
+      ) : (
+        <div
+          css={`
+            text-align: center;
+          `}
+        >
+          <div
+            css={`
+              ${textStyle('title4')};
+            `}
+          >
+            Your delegate
+          </div>
+          <div
+            css={`
+              margin-top: ${2 * GU}px;
+            `}
+          >
+            {supporter?.representative ? (
+              <Representative
+                onRemoveDelegate={onRemoveDelegate}
+                onSetDelegate={onSetDelegate}
+                representative={supporter.representative}
+              />
+            ) : (
+              <div
+                css={`
+                  text-align: center;
+                `}
+              >
+                <div
+                  css={`
+                    ${textStyle('body1')};
+                  `}
+                >
+                  None
+                </div>
+                <span
+                  css={`
+                    ${textStyle('body3')};
+                    color: ${theme.contentSecondary};
+                  `}
+                >
+                  (You vote for yourself)
+                </span>
+                <Button
+                  mode="strong"
+                  label="Delegate voting"
+                  wide
+                  onClick={onSetDelegate}
+                  css={`
+                    margin-top: ${3 * GU}px;
+                  `}
+                />
+              </div>
+            )}
+          </div>
+        </div>
+      )}
+    </Box>
+  )
+}
+
+function Representative({ onRemoveDelegate, onSetDelegate, representative }) {
+  const theme = useTheme()
+  const history = useHistory()
+  const profile = useProfile(representative.address)
+
+  const handleViewProfile = useCallback(() => {
+    history.push(`/profile?account=${representative.address}`)
+  }, [history, representative])
+
+  return (
+    <div>
+      <div
+        onClick={handleViewProfile}
+        css={`
+          margin-bottom: ${1 * GU}px;
+          cursor: pointer;
+        `}
+      >
+        {profile?.image ? (
+          <img
+            src={profile.image}
+            height="72"
```

Comment:
let's use the GU

Reply:
We have agreed in the past that we´ll use GU only for margins, paddings and other spacing styles but not for element dimension.